config/deploy/staging.rb

```diff
@@ -1,10 +1,19 @@
-# using an ssh-config entry to set the user, key, and address of the machine
+#Using the aws-sdk to locate servers via the Role tag and deploy to them.
+require 'aws-sdk-ec2'
 set :stage, :staging
 set :rails_env, 'production'
-server 'kithe.sciencehistory.org', user: 'digcol', roles: [:web, :app, :db, :jobs, :solr, :cron]
-#server '34.227.26.131', user: 'hydep', roles: [:solr]
-#server '54.204.60.1', user: 'hydep', roles: [:jobs, :cron] # dzi bg job box
-
+#Edit ec2 for region changes, right now we only use one region.
+ec2 = Aws::EC2::Resource.new(region:'us-east-1')
+#Server role keys should be the Role tag (assigned by ansible) that you want to deploy to. The array value is the list of capistrano roles that the server needs.
+server_roles = {"kithe"=>[':web', ':app', ':db', ':jobs', ':solr', ':cron']}
+server_roles.each do |key, value|
+  current_server_roles = "#{value}"
```

Comment:
I don't think there's a reason to take `value` and put it in a new variable as `"#{value}"`?  If you want to call the variable `current_server_roles`, just call it that above in `do |key, current_server_roles|`, I think?  (although I think ti's always _singular_ so should maybe be `current_server_role`?)